README.md

```diff
@@ -4,41 +4,185 @@ Please edit this README.md file with your answers to the worksheet questions.
+|F | T | F |   F   |       F      |    F     |            F             |
+|F | T | T |   F   |       F      |    T     |            F             |
+|T | F | F |   T   |       T      |    T     |            T             |
+|T | F | T |   T   |       T      |    T     |            T             |
+|T | T | F |   F   |       T      |    T     |            T             | 
+|T | T | T |   F   |       T      |    T     |            T             |
+
+### d 
+|B | D | NOT C | NOT A | NOT (B OR NOT C) | (NOT A AND D) | NOT (B OR NOT C) AND (NOT A AND D)|A AND NOT (B OR NOT C) AND (NOT A AND D)|
+|:-|:-:|:-----:|:-----:|:----------------:|:-------------:|:---------------------------------:| --------------------------------------:|
+|F | F |   T   |  T    |        F         |       F       |                 F                 |                   F                    |
+|F | T |   T   |  T    |        F         |       T       |                 F                 |                   F                    |
+|F | F |   F   |  T    |        T         |       F       |                 F                 |                   F                    |
+|F | T |   F   |  T    |        T         |       T       |                 T                 |                   F                    |
+|T | F |   T   |  T    |        F         |       F       |                 F                 |                   F                    |
+|T | T |   T   |  T    |        F         |       T       |                 F                 |                   F                    |
+|T | F |   F   |  T    |        F         |       F       |                 F                 |                   F                    |
+|T | T |   F   |  T    |        F         |       T       |                 F                 |                   F                    |
+|F | F |   T   |  F    |        F         |       F       |                 F                 |                   F                    |
+|F | T |   T   |  F    |        F         |       F       |                 F                 |                   F                    |
+|F | F |   F   |  F    |        T         |       F       |                 F                 |                   F                    |
+|F | T |   F   |  F    |        T         |       F       |                 F                 |                   F                    |
+|T | F |   T   |  F    |        F         |       F       |                 F                 |                   F                    |
+|T | T |   T   |  F    |        F         |       F       |                 F                 |                   F                    |
+|T | F |   F   |  F    |        F         |       F       |                 F                 |                   F                    | 
+|T | T |   F   |  F    |        F         |       F       |                 F                 |                   F                    |
 
 ## Question 2
```

Comment:
It looks like you didn't upload your diagrams with your solution so I can't see them!